com.unity.mobile.notifications/Documentation~/Android.md

```diff
@@ -49,18 +49,20 @@ Android recommends to not use exact scheduling due to higher power consumption i
 
 ## Request permission to post notifications
 
-Starting with Android 13.0 (API level 33) notifications can not be posted without users permission. They can still be scheduled, but will work silently with no UI shown to the user. You can request the permission by running this method in the coroutine:
+Starting with Android 13.0 (API level 33) notifications cannot be posted without user's permission. They can still be scheduled, but will work silently with no UI shown to the user. You can request the permission by running this method in the [coroutine](https://docs.unity3d.com/6000.0/Documentation/Manual/Coroutines.html):
 
 ```c#
 IEnumerator RequestNotificationPermission()
 {
     var request = new PermissionRequest();
     while (request.Status == PermissionStatus.RequestPending)
         yield return null;
-    // here use request.Status to determine users response
+    // here use request.Status to determine user's response
 }
 ```
 
+The coroutine allows you to asynchronously wait until the user responds to the permission request. You can use `request.Status` to check whether the user has granted or denied the permission and proceed with posting notifications accordingly. 
```

Comment:
Added these couple sentences. Is there anything that would be useful to mention here?

Reply:
Can't think of more. Nice addition.